hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/PolicyState.java

```diff
@@ -44,11 +47,14 @@
     TIMEOUT(TimeoutState.class),
     MATCHING_SERVICE_REQUEST_ERROR(MatchingServiceRequestErrorState.class),
     USER_ACCOUNT_CREATION_REQUEST_SENT(UserAccountCreationRequestSentState.class),
+    EIDAS_USER_ACCOUNT_CREATION_REQUEST_SENT(EidasUserAccountCreationRequestSentState.class),
     AUTHN_FAILED_ERROR(AuthnFailedErrorState.class),
+    COUNTRY_AUTHN_FAILED_ERROR(CountryAuthnFailedErrorState.class),
     FRAUD_EVENT_DETECTED(FraudEventDetectedState.class),
     REQUESTER_ERROR(RequesterErrorState.class),
     CYCLE_3_DATA_INPUT_CANCELLED(Cycle3DataInputCancelledState.class),
-    USER_ACCOUNT_CREATION_FAILED(UserAccountCreationFailedState.class);
+    USER_ACCOUNT_CREATION_FAILED(UserAccountCreationFailedState.class),
+    COUNTRY_USER_ACCOUNT_CREATION_FAILED(CountryUserAccountCreationFailedState .class);
```

Comment:
Same - The enum name can still be eIDAS though, no? (Even if the state class starts with Country)

Reply:
@IreneLau-GDS @CharlesIC I think we should agree on a convention - either EIDAS or Country and stick with it - personally I think Country is less jargon-y?

Reply:
after discussion - we're settling on using EIDAS for consistency with configurations - Renaming can be done as separate PR

Reply:
Story for renaming: https://govukverify.atlassian.net/browse/EID-919